Assets/HoloToolkit/Input/Scripts/Cursor/ObjectCursor.cs

```diff
@@ -51,21 +51,19 @@ public override void OnCursorStateChange(CursorStateEnum state)
 
             if (state != CursorStateEnum.Contextual)
             {
-                // Hide all children first
-                for(int i = 0; i < ParentTransform.childCount; i++)
+                var newActive = CursorStateData.FirstOrDefault(p => p.CursorState == state);
```

Comment:
Oh. I almost never do that. Thank ReSharper for learning me that. Do you want me to change it, or are you just going to merge?